src/main/java/org/folio/service/mapping/processor/rule/Rule.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.service.mapping.processor.rule;
+
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+public class Rule {
+  private List<DataSource> dataSources = new ArrayList<>();
+
+  public Rule(JsonObject rule) {
```

Comment:
@KVupp    this converting logic from JSON object to POJO  is customized. There is a need to pass tag to datasource, I don't know how to convert json object in other was

Reply:
I added a branch to see how it can be made a little simpler: https://github.com/folio-org/mod-data-export/pull/55